examples/models/contrib/chatglm-6b/README.md

```diff
@@ -0,0 +1,106 @@
+# ChatGLM
```

Comment:
I suggest we merge chatglm-6b, chatglm2-6b and chatglm3-6b-32k into one directory `examples/models/contrib/chatglm`

Reply:
I think splitting them is better. Otherwise, it is confused why glm_4 is not covered in chatglm.